NitroxPatcher/Patches/Dynamic/CellManager_RegisterEntity_Patch.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Reflection;
+using Harmony;
+using NitroxClient.GameLogic;
+using NitroxModel.Core;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    public class CellManager_RegisterEntity_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Do we need this?

Reply:
Currently not. It is/was for tracking, where and when a Base and/or a BaseRoot Object is generated from the game. The RegisterEntity patch may also be useful to analyse spawn issues, because this gets triggered for every entity in the game.